integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -277,6 +312,18 @@ def clean(self):
         ):
             cleaned_data["deepl_midyear_start_month"] = None
 
+        # Make sure no non-disableable offers have been disabled
+        if (
+            self.disabled_offer_options
+            and not all(self.disabled_offer_options) in cleaned_data["offers"]
```

Comment:
```suggestion
            and not all(offer in cleaned_data["offers"] for offer in self.disabled_offer_options)
```
Maybe this is what you intended?